App/App_macOS.swift

```diff
@@ -21,7 +21,7 @@ import CoreKiwix
 
 #if os(macOS)
 final class AppDelegate: NSObject, NSApplicationDelegate {
-    @MainActor func applicationShouldTerminateAfterLastWindowClosed(_ sender: NSApplication) -> Bool {
+    func applicationShouldTerminateAfterLastWindowClosed(_ sender: NSApplication) -> Bool {
```

Comment:
Here the runtime was complaining that it is not performant to execute this function on the main thread.